cogs/helpers.py

```diff
@@ -53,11 +56,57 @@
 CURRENCY_TEMPLATE = "http://www.xe.com/currencyconverter/convert/" \
                     "?Amount={}&From={}&To={}"
 
+try:
```

Comment:
This should be factored into the initialization of a separate cog for this functionality. (Another reason for not mixing this with the rest of helper functionality).

Reply:
This will be addressed in a future PR once we have more subscription-based functions that aren't called by commands.